cnames_active.js

```diff
@@ -424,6 +424,7 @@ var cnames_active = {
   "corki": "dvtate.github.io/corki",
   "corona": "soroushchehresa.github.io/awesome-coronavirus",
   "corps": "copay.github.io",
+  "classroom: "epoxydevelopment.github.io/Classroom",
```

Comment:
```suggestion
  "classroom": "epoxydevelopment.github.io/Classroom",
```
Please fix this formatting issue

Reply:
Ohh okay i will fix it